### ISSUE TITLE
[BUG] when selecting text then replacing new text

### ISSUE THREAD

**Issue by bolaxd** [deadlyjack/Acode#810]
I'm not familiar with acode version 1.8.6, 
and two versions before it.  

I have experienced 2 updates before, and in version 1.8.4 I have difficulty writing code. 

 I don't know if this is a bug or just an additional feature but it's not finished.  

but I think this is a bug, because for me this is an oddity.

I use HP Oppo A53, Android 12 [Acode Premium], Please fix this BUG immediately 👍 

[Video Detail BUG](https://telegra.ph/file/61aed220e0ce9a9df4e49.mp4)

**Comment by asl97** [comment]
you can go into the setting and change keyboard mode to `no suggestions` or learn to properly end the keyboard suggestion with a space, symbol, enter or by pressing the word for now

This seem to be an issue with keyboard suggestion, there also other weirdness with it like the following.

tapping on 'randomish' part of the editor and pressing a button
https://github.com/deadlyjack/Acode/assets/4752645/d04a19cb-a449-414e-b7cb-e8e036c429db

alternating between enter and backspace
https://github.com/deadlyjack/Acode/assets/4752645/b0904ae4-680e-4821-b8f0-96ffb7493a17

**Comment by bolaxd** [comment]
> you can go into the setting and change keyboard mode to `no suggestions` or learn to properly end the keyboard suggestion with a space, symbol, enter or by pressing the word for now
> 
> This seem to be an issue with keyboard suggestion, there also other weirdness with it like the following.
> 
> tapping on 'randomish' part of the editor and pressing a button
> 
>  Screenrecorder-2023-08-31-06-42-17-478.mp4 
> alternating between enter and backspace
> 
>  Screenrecorder-2023-08-31-06-52-10-404.mp4

wow, thanks for helping with this problem, I hope this Acode Editor will continue to develop well and can help someone who doesn't have a computer

**Comment by asl97** [comment]
I don't think this should be closed as this is a fairly buggy behavior that still exists

@deadlyjack I believe the following behavior would be desirable
  1. completion/suggestion should ends on touch/input
  2. when backspace delete a newline, it shouldn't try to do completion/suggestion

**Comment by bolaxd** [comment]
> 

Does that mean this is a fairly serious bug? I'll help reopen this issue so that other developers can hear or listen to it
